pep-0012.rst

```diff
@@ -475,68 +475,71 @@ Footnotes containing the URLs from external targets will be generated
 automatically at the end of the References section of the PEP, along
 with footnote references linking the reference text to the footnotes.
 
-Text of the form "PEP x" or "RFC x" (where "x" is a number) will be
-linked automatically to the appropriate URLs.
+To refer to PEPs or RFCs, always use the ``:pep:`` and ``:rfc:`` roles,
+never hard-coded URLs.
```

Comment:
I would have written it like this (11 examples already in PEPs), but fine if you prefer the original (7 in PEPs):
```suggestion
never hardcoded URLs.
```